zboxcore/sdk/common_test.go

```diff
@@ -0,0 +1,393 @@
+package sdk
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/0chain/gosdk/zboxcore/blockchain"
+	"github.com/0chain/gosdk/zboxcore/sdk/mocks"
+	"github.com/stretchr/testify/assert"
+
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	configDir            = "testdata"
+	syncTestDir          = configDir + "/" + "sync"
+	syncDir              = syncTestDir + "/" + "sync_alloc"
+	textPlainContentType = "text/plain"
+)
+
+func blobberIDMask(idx int) string {
+	return fmt.Sprintf("${blobber_id_%v}", idx)
+}
+
+func blobberURLMask(idx int) string {
+	return fmt.Sprintf("${blobber_url_%v}", idx)
+}
+
+func parseFileContent(t *testing.T, fileName string, jsonUnmarshalerInterface interface{}) (fileContentBytes []byte) {
+	fs, err := os.OpenFile(fileName, os.O_RDONLY, 0644)
+	assert.NoErrorf(t, err, "Error os.OpenFile() %v: %v", fileName, err)
+
+	defer fs.Close()
+	bytes, err := ioutil.ReadAll(fs)
+	assert.NoErrorf(t, err, "Error ioutil.ReadAll() cannot read file content of %v: %v", fileName, err)
+	if jsonUnmarshalerInterface != nil {
+		err = json.Unmarshal(bytes, jsonUnmarshalerInterface)
+		assert.NoErrorf(t, err, "Error json.Unmarshal() cannot parse file content to %T object: %v", jsonUnmarshalerInterface, err)
+	}
+
+	return bytes
+}
+
+func writeFileContent(t *testing.T, fileName string, fileContentBytes []byte) {
+	fs, err := os.OpenFile(fileName, os.O_RDWR|os.O_CREATE, 0644)
+	assert.NoErrorf(t, err, "Error os.OpenFile() %v: %v", fileName, err)
+	defer fs.Close()
+	_, err = fs.Write(fileContentBytes)
+	assert.NoErrorf(t, err, "Error fs.Write() cannot write file content to %v: %v", fileName, err)
+}
+
+type nodeConfig struct {
+	BlockWorker       string   `yaml:"block_worker"`
+	PreferredBlobbers []string `yaml:"preferred_blobbers"`
+	SignScheme        string   `yaml:"signature_scheme"`
+	ChainID           string   `yaml:"chain_id"`
+}
+
+func setupMockInitStorageSDK(t *testing.T, configDir string, blobberNums int) (miners, sharders []string, blobbers []*mocks.Blobber, close func()) {
+	var nodeConfig *nodeConfig
+
+	nodeConfigBytes := parseFileContent(t, configDir+"/"+"config.yaml", nil)
+	err := yaml.Unmarshal(nodeConfigBytes, &nodeConfig)
+	assert.NoErrorf(t, err, "Error yaml.Unmarshal(): %v", err)
+
+	clientBytes := parseFileContent(t, configDir+"/"+"wallet.json", nil)
+	clientConfig := string(clientBytes)
```

Comment:
There is no need to load clientConfig data from the file system. Just defie a lcoal clientConfig string. If it changes from test to test include it in `setupMockInitStorageSDK` input parameters somewhere.